charts/firefly/templates/erc20/deployment.yaml

```diff
@@ -0,0 +1,109 @@
+{{/*
+  Copyright © 2022 Kaleido, Inc.
+
+  SPDX-License-Identifier: Apache-2.0
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+  http://swww.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/}}
+
+{{- if .Values.erc20.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "firefly.fullname" . }}-erc20
+  labels:
+  {{- include "firefly.erc20Labels" . | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      {{- include "firefly.erc20SelectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.erc20.podAnnotations }}
+      annotations:
+      {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+    {{- include "firefly.erc20SelectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.erc20.imagePullSecrets }}
+      imagePullSecrets:
+      {{- toYaml . | nindent 8 }}
+      {{- end }}
+      securityContext:
+      {{- toYaml .Values.erc20.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}-erc20
+          securityContext:
+          {{- toYaml .Values.erc20.securityContext | nindent 12 }}
+          image: "{{ .Values.erc20.image.repository }}:{{ .Values.erc20.image.tag }}"
+          imagePullPolicy: {{ .Values.erc20.image.pullPolicy }}
+          ports:
+            - name: http
+              containerPort: {{ .Values.erc20.service.port }}
+              protocol: TCP
+          livenessProbe:
+            httpGet:
+              path: /api/v1/health/liveness
```

Comment:
We'll need to update ERC20 to include these endpoints like we did for ERC1155. Will get a PR open for that as I test this.

Reply:
Opened https://github.com/hyperledger/firefly-helm-charts/issues/11 instead as a follow up.